openedx/core/djangoapps/schedules/tests/test_signals.py

```diff
@@ -178,7 +178,7 @@ def test_not_updated_since_new_start_in_past(self, mock_get_current_site):
         course.start = course.start + datetime.timedelta(days=-10)  # new course start changes to a past date
         self.store.update_item(course, ModuleStoreEnum.UserID.test)
         enrollment = CourseEnrollment.objects.get(id=enrollment.id)
-        self.assert_schedule_dates(enrollment.schedule, previous_start)  # start remains unchanged
+        self.assert_schedule_dates(enrollment.schedule, course.start)  # start set to new course start
```

Comment:
Can you update the test method names for both of these tests since the expected behavior has now changed?

Reply:
Good catch, fixed!